cypress/locators/pager/index.js

```diff
@@ -7,6 +7,7 @@ export const pagerSummary = () => cy.iFrame(PAGER_SUMMARY)
   .find('div:nth-child(3)');
 export const pageSelect = () => cy.iFrame(PAGE_SELECT);
 export const maxPages = () => cy.iFrame(MAX_PAGES);
-export const pageInput = () => cy.iFrame(PAGE_INPUT);
+export const currentPageInputClassic = () => cy.iFrame(PAGE_INPUT).find('input');
+export const currentPageInput = () => cy.iFrame(PAGE_INPUT).find('input');
```

Comment:
what is the difference between those 2 locators?

Reply:
I've deleted one of them